### PR TITLE
Shivani | Test | Show connect bank and open bank list at dashboard and at ledger page

### PR DESCRIPTION
**Pull request by shivanivani02**
https://app.clickup.com/t/86cx3fz91

## Summary by CodeRabbit

- **New Features**
	- Introduced a new `BankIntegrationComponent` for managing bank integrations.
	- Added a new `InstitutionsListComponent` for displaying banking institutions.
	- Enhanced the user interface with buttons for connecting bank accounts and improved localization support.
	- Implemented a new `<bank-integration>` component to streamline payment integration.

- **Bug Fixes**
	- Corrected import paths for `SettingsIntegrationService` across multiple components.

- **Documentation**
	- Updated localization files with new keys for bank connectivity in English, Hindi, and Marathi. 

- **Style**
	- Enhanced styling for the bank integration component for better user experience. 

- **Tests**
	- Improved state management and error handling in the bank integration functionalities.

### CODE REVIEW

File: apps/web-giddh/src/app/home/components/bank-accounts/bank-accounts.component.ts [hunk 1]
```diff
@@ -32,12 +36,22 @@ export class BankAccountsComponent implements OnInit, OnDestroy {
...
+    public hasIntegrationScope: boolean = false;
+    
     constructor(
         private store: Store<AppState>,
         private contactService: ContactService,
         private commonAction: CommonActions,
-        private changeDetectionRef: ChangeDetectorRef
+        private changeDetectionRef: ChangeDetectorRef,
+        public dialog: MatDialog,
+        private generalService: GeneralService,
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/home/components/bank-accounts/bank-accounts.component.ts:55]:
```suggestion
        private generalService: GeneralService
```

File: apps/web-giddh/src/app/home/components/bank-accounts/bank-accounts.component.ts [hunk 2]
```diff
@@ -97,9 +123,34 @@ export class BankAccountsComponent implements OnInit, OnDestroy {
...
+    /**
+    * This function will use for get institutions details
+    *
+    * @param {*} element
+    * @memberof BankAccountsComponent
+    */
+    public openInstitutionsDialog(): void {
+        let data = {
+            localeData: this.localeData,
+            commonLocaleData: this.commonLocaleData,
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/home/components/bank-accounts/bank-accounts.component.ts]:
```suggestion
            commonLocaleData: this.commonLocaleData
```

File: apps/web-giddh/src/app/home/home.module.ts [hunk 3]
```diff
@@ -37,7 +38,7 @@ import { MatIconModule } from '@angular/material/icon';
         TotalOverduesChartComponent,
         ProfitLossComponent,
         BankAccountsComponent,
-        CrDrComponent
+        CrDrComponent,
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/home/home.module.ts]:
```suggestion
        CrDrComponent
```

File: apps/web-giddh/src/app/ledger/ledger.component.ts [hunk 4]
```diff
@@ -2587,12 +2636,15 @@ export class LedgerComponent implements OnInit, OnDestroy {
      */
     public translationComplete(event: boolean): void {
         if (event) {
+
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/ledger/ledger.component.ts]: remove extra line

**Review by dilpreetsinghofficial — CHANGES_REQUESTED** [review]

**Review by dilpreetsinghofficial — APPROVED** [review]

File: apps/web-giddh/src/app/ledger/ledger.component.ts [hunk 5]
```diff
@@ -312,6 +314,14 @@ export class LedgerComponent implements OnInit, OnDestroy {
     private bankTransactionsWithAccountName: any[] = [];
     /** True if consolidated branch */
     public isConsolidatedBranch: boolean;
+    /** Hold reference number */
+    public referenceNumber: string = '';
+    /** True if api call in progress */
+    public isLoading: boolean = false;
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/ledger/ledger.component.ts:322]: remove this unused variable

**Review by dvCodeWorld — APPROVED** [review]

**Pull request merged**